util/threading.h

```diff
@@ -135,8 +136,11 @@ class PerClusterPools {
       inner_pools_.push_back(std::make_unique<hwy::ThreadPool>(num_threads));
       if (num_threads > 1) {
         inner_pools_.back()->Run(0, num_threads,
-                                 [](uint64_t /*task*/, size_t thread) {
-                                   hwy::PinThreadToLogicalProcessor(thread);
+                                 [pin_offset, num_threads](uint64_t /*task*/,
```

Comment:
This logic seems reasonable, but it only runs if topology detection fails. Are you running on Linux? If so, this is probably not reached.

Reply:
My main platform is Linux, but I also run it on macOS. :)

Reply:
Got it :)